user-guide/Advanced_Modules/Dashboards_app/Using_dashboard_feeds/Specifying_data_input_in_a_dashboard_URL.md

```diff
@@ -6,64 +6,117 @@ uid: Specifying_data_input_in_a_dashboard_URL
+  ```json
+  {
+  "cid": <component-id>,
+  "select": <data>
+  }
+  ```
 
   > [!NOTE]
-  > From DataMiner 10.2.0/10.1.1 onwards, for a predefined time span in a time range component, the URL contains the name of that time span (e.g. today, yesterday, etc.) instead of the start time and end time of the time span.
+  > You can find the ID of each component in the lower right corner of the component while in edit mode.
 
-- Bookings (using the argument *bookings*): By specifying the booking ID. Supported from DataMiner 10.0.3 onwards.
+## Legacy syntax
 
-- Service definitions (using the argument *service definitions*): By specifying the service definition ID(s). Supported from DataMiner 10.0.5 onwards.
+Prior to DataMiner 10.2.0/10.2.2, you can pass data to a dashboard using the following URL syntax:
 
-- An EPM filter (using the argument *cpes*): By specifying the DMA ID, the element ID, the field PID, the table PID and the value. From DataMiner 10.0.5 onwards, you should instead specify the DMA ID, the element ID, the field PID, the field value, the table index PID and the index value.
+``url?<datatype1>=<datakeys1>&<datatype2>=<datakeys2>&...``
 
-  > [!NOTE]
-  > - If, in the time spans argument, you specify “/1531814522191” (i.e. leaving out the first time stamp), this will be interpreted as “from midnight to X”.
-  > - If, in the time spans argument, you specify “1531810522191” (i.e. leaving out the second time stamp), this will be interpreted as “from X to now”.
+In the syntax above, "datatype" is one of the objects mentioned below (e.g. "elements"), and "datakeys" is its identifier (e.g. the DMA ID and element ID).
 
 Within one object, use a slash (“/”) as the separator between its components. If different objects of the same type are specified, use “%1D” as the separator between the objects.
 
 For example:
 
-- ``http://myDma/Dashboard/#/myDashboard?elements=1/1%1D1/2&views=1&parameters=1/1/1%1D1/1/2/myIndex``
+- ``https://myDma/Dashboard/#/myDashboard?elements=1/1%1D1/2&views=1&parameters=1/1/1%1D1/1/2/myIndex``
 
   This URL opens a dashboard in which the elements 1/1 and 1/2, view 1 and parameters 1/1/1 and 1/1/2/myIndex are selected by default.
 
-- ``http://myDma/Dashboard/#/myDashboard?time%20spans=1549753200000/1549835265007``
+- ``https://myDma/Dashboard/#/myDashboard?time%20spans=1549753200000/1549835265007``
 
   This URL opens a dashboard with a time range filter from 1549753200000 to 1549835265007.
 
 > [!NOTE]
-> - Additional URL options are possible from DataMiner 10.0.2 onwards. To only display a dashboard, without the rest of the app, add the argument “*embed=true*”. To display the *Clear all* button for an embedded dashboard, add “*subheader=true*” as well. For example: *http://**\[DMA IP\]**/dashboard/#/MyDashboards/dashboard.dmadb?embed=true&subheader=true*
+> - This syntax continues to be supported in recent DataMiner versions.
```

Comment:
Maybe explicitly mention that it continues to be supported for backwards compatibility reasons (i.e. making sure existing urls won't break) but that it is recommended to use the json syntax from now on for more flexibility.

At some point in the future we might be forced to drop support for the legacy syntax.